src/stanExports_estimate_infections.h

```diff
@@ -39,17 +39,17 @@ stan::io::program_reader prog_reader__() {
     reader.add_event(14, 2, "restart", "model_estimate_infections");
     reader.add_event(14, 2, "include", "functions/discretised_lognormal_pmf.stan");
     reader.add_event(14, 0, "start", "functions/discretised_lognormal_pmf.stan");
-    reader.add_event(23, 9, "end", "functions/discretised_lognormal_pmf.stan");
```

Comment:
I am not familiar with c++ so can't meaningfully review these

Reply:
These are automated files built from `stan` code and no problem.